### ISSUE TITLE
request: update Timepicker Addon to v1.6.1

### ISSUE THREAD

**Issue by outflux3** [ryancramerdesign/ProcessWire#1522]
the latest version supports selects for time; perhaps time to update this?

**Comment by LostKobrakai** [comment]
Or rather change to something like https://github.com/rometools/rome as datepickers are one of really few things in pw that rely on jquery UI, which is quite a package just for a datepicker.

**Comment by apeisa** [comment]
I think jQuery UI is used quite a bit. Modals, sortables, autocomplete at
least. Also something inputfield developers can count on.
24.11.2015 19.19 "Benjamin Milde" <notifications@github.com> kirjoitti:

**Comment by LostKobrakai** [comment]
Yeah it was FormBuilder where this did annoy me, not the admin backend.

**Comment by ryancramerdesign** [comment]
I have updated to the latest timepicker for the devns branch (will be committed soon). Thanks for letting me know, lots of good new options, especially for localization (which the previous one lacked).

**Comment by outflux3** [comment]
great! yeah i've been using the select/one line mode with 5 or 15 min increments based on client request and it is better than the sliders; i18n will also be helpful!
